### PR TITLE
[CI:DOCS] extend docs to include help for when pub/priv key is signing issue

### PR DESCRIPTION
**Pull request by gigatexal**
extend docs to include help for when pub/priv key is signed with an unsupported algo

### CODE REVIEW

**Comment by gigatexal** [comment]
I ran into an issue on a fresh install of fedora 34. It wouldn't accept my rsa-sha signed key. I have no idea if this is ok how I've added the docs -- I am not a technical writer -- so I am totally open to feedback.

Also, not quite sure if this should be a seperate file that is linked to the original one? Is it too specific talking about Fedora?

/assign @rhatdan

**Comment by gigatexal** [comment]
Thinking about this some more -- I should probably add this to the troubleshooting doc but I do think it wouldn't hurt to have a link to that document in this one as well maybe under a Troubleshooting header.

**Comment by rhatdan** [comment]
You need to sign your commits, and to make sure the description is less then 80 Chars followed by a blank line.

git commit -a --amend -s
git push --force

File: docs/tutorials/remote_client.md [hunk 1]
```diff
@@ -108,5 +108,25 @@ podman-remote system connection --help
...
 
+# Troubleshooting
+
+If after adding the remote system via ssh uri you get an error similar to the following:
+
+```
+Error: failed to create sshClient: Connection to bastion host (ssh://user@host:22/run/user/.../podman/podman.sock) failed.: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
+```
+
+and you are running a recent version of Fedora Server it could be that the key you're using was not created with an allowed algorithm.
```
Comment (rhatdan) [inline docs/tutorials/remote_client.md:119]: Do not mention Fedora here.   Remove `and you are running a recent version of Fedora Server`

**Review by rhatdan — APPROVED** [review]
I think most of this text would be better in troubleshooting.md.  With a smaller section in the man page saying to check your ssh configuration.

**Comment by gigatexal** [comment]
@rhatdan I added the fixes

**Comment by jwhonce** [comment]
LGTM

**Comment by gigatexal** [comment]
@rhatdan how do I pass the "lgtm" label to the tide bot?

`tide Pending — Not mergeable. Needs lgtm label.`

**Comment by mheon** [comment]
We require two maintainer LGTMs to merge. After the first LGTM, another maintainer will review and, if they want to merge, command the bot proceed. For example:

/lgtm

**Pull request merged**